rails/app/views/dashboard/communities/show.html.erb

```diff
@@ -76,3 +118,15 @@
 
   <%= f.submit %>
 <% end %>
+
+<script>
+  function copyText(e) {
+    e.preventDefault()
+    navigator.clipboard.writeText(this.innerText.trim())
+  }
+
+  let boxes = document.getElementsByClassName('copiableText')
+  for(const t of boxes) {
+    t.addEventListener('click', copyText)
+  }
```

Comment:
We could, but it won't actually copy the contents of multiple divs. Each instance of `copiableText` receives its own event listener handler, so it would only copy its own internal instances innerText. It allows us to have multiple copiableText boxes on one page (even though we don't use it more than once).

Reply:
oh if it's not copying the contents of multiple divs it's fine as is